src/pl-index.c

```diff
@@ -2046,7 +2046,7 @@ listIndexGenerations(Definition def, gen_t gen)
 	    for(cr=cl->first_clause; cr; cr=cr->next)
 	    { Clause clause = cr->value.clause;
 
-	      Sdprintf("  %p: [%2d] %8u-%10u%s%s\n",
+	      Sdprintf("  %p: [%2d] %8" PRIuMAX "-%10" PRIuMAX "%s%s\n",
```

Comment:
Do we have support for PRIuMAX in Sdprintf?    Better yet, use generationName(), which was added when generations became sometimes huge numbers when in a transaction.

Reply:
Done. Please check that this is the correct way to call generationName() -- it appears to allocate something on the stack and I want to ensure that this is a valid thing to do here. (also at line 2060 and pl-proc.c:4118)